lxl-web/src/routes/(app)/[[lang=lang]]/(find)/[fnurgel=fnurgel]/InstancesList.svelte

```diff
@@ -212,4 +214,8 @@
 	details[open] > summary {
 		@apply bg-pill/8 hover:bg-pill/16;
 	}
+
+	details[open] > summary .arrow {
+		@apply rotate-90 transition-transform;
```

Comment:
this removes the transition class when the panel's not open, resulting in no closing animation.
suggestion:
```
.arrow {
  @apply rotate-0 transition-transform;
}

[open] > .arrow {
  @apply rotate-90;
}
```